extract/dcinside/main.py

```
@@ -286,33 +273,37 @@ def get_html_of_post(self, driver:webdriver.Chrome, url:str):
                     return soup
             
             except:# 페이지 접근 재시도
```

Ruff (0.8.2)

Do not use bare `except`

(E722)